File: app/src/main/java/com/tapura/podmorecasts/model/Episode.java
package com.tapura.podmorecasts.model;

public class Episode {

    private String title;
    private String link;
    private String episodeLink;
    private String description;
    private String pathInDisk;
    private EpisodeMediaState episodeState;

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getLink() {
        return link;
    }

    public void setLink(String link) {
        this.link = link;
    }

    public String getEpisodeLink() {
        return episodeLink;
    }

    public void setEpisodeLink(String episodeLink) {
        this.episodeLink = episodeLink;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public String getPathInDisk() {
        return pathInDisk;
    }

    public void setPathInDisk(String pathInDisk) {
        this.pathInDisk = pathInDisk;
    }

    public EpisodeMediaState getEpisodeState() {
        return episodeState;
    }

    public void setEpisodeState(EpisodeMediaState episodeState) {
        this.episodeState = episodeState;
    }
}

File: app/src/main/java/com/tapura/podmorecasts/model/Episode.kt
package com.tapura.podmorecasts.model


const val STATE_NOT_IN_DISK = 0
const val STATE_DOWNLOADING = 1
const val STATE_DOWNLOADED = 2
data class Episode(
        var title: String = "",
        var link: String = "",
        var episodeLink: String = "",
        var description: String = "",
        var pathInDisk: String? = null,
        var guid: String = "",
        var episodeState: Int = STATE_NOT_IN_DISK
)